pkg/dockerregistry/client.go

```diff
@@ -19,10 +19,23 @@ import (
 	imageapi "github.com/openshift/origin/pkg/image/api"
 )
 
+const (
+	publicV1DockerHubHost = "index.docker.io"
+	// TODO: is there a better URL?
+	publicV2DockerHubHost = "registry-1.docker.io"
```

Comment:
We try not to import docker constants because they're usually poorly scoped.  docker/registry brings in a massive chain of dependencies (or used to).